_episodes/04-history.md

```diff
@@ -20,41 +20,50 @@ As we saw in the previous lesson, we can refer to commits by their
 identifiers.  You can refer to the _most recent commit_ of the working
 directory by using the identifier `HEAD`.
 
-We've been adding one line at a time to `mars.txt`, so it's easy to track our
+We've been adding one line at a time to `README.md`, so it's easy to track our
 progress by looking, so let's do that using our `HEAD`s.  Before we start,
-let's make a change to `mars.txt`.
+let's make a change to `README.md`.
 
 ~~~
-$ nano mars.txt
-$ cat mars.txt
+$ nano README.md
+$ cat README.md
 ~~~
 {: .bash}
 
 ~~~
-Cold and dry, but everything is my favorite color
-The two moons may be a problem for Wolfman
-But the Mummy will appreciate the lack of humidity
-An ill-considered change
+# about your new image collection
+
+The `cats-human-situations.csv` file contains metadata for three image objects. The original metadata from the source institutions has been abbreviated and made messier so you have something to clean up!
+
+Image credits:
+- The black cat, March 1896 (http://ark.digitalcommonwealth.org/ark:/50959/w0892s818). Courtesy Boston Public Library via Digital Commonwealth.
+- Puzzums in costume (http://photos.lapl.org/carlweb/jsp/DoSearch?index=z&databaseID=968&terms=0000068045). Courtesy Los Angeles Public Library.
+- A Xmas shopper (http://digitalcollections.nypl.org/items/510d47e1-43b5-a3d9-e040-e00a18064a99). Courtesy New York Public Library.
+
+The images are in the `images/` subdirectory of this repository.
+If you would like, you can also put images of dogs and hexagonal software logos here.
+
 ~~~
 {: .output}
 
 Now, let's see what we get.
 
 ~~~
-$ git diff HEAD mars.txt
+$ git diff HEAD README.md
 ~~~
 {: .bash}
 
 ~~~
-diff --git a/mars.txt b/mars.txt
-index b36abfd..0848c8d 100644
---- a/mars.txt
-+++ b/mars.txt
-@@ -1,3 +1,4 @@
- Cold and dry, but everything is my favorite color
- The two moons may be a problem for Wolfman
- But the Mummy will appreciate the lack of humidity
-+An ill-considered change.
+diff --git a/README.md b/README.md
+index 17e3cdd..0e44dff 100644
+--- a/README.md
++++ b/README.md
+@@ -8,3 +8,4 @@ Image credits:
+ - A Xmas shopper (http://digitalcollections.nypl.org/items/510d47e1-43b5-a3d9-e040-e00a18064a99). Courtesy New York Public Library.
+ 
+ The images are in the `images/` subdirectory of this repository.
++If you would like, you can also put pictures of dogs and hexagonal software logos here.
+
 ~~~
 {: .output}
```

Comment:
Everything above here is already covered in the previous section, I think.

Reply:
true. I think the repetition is intentional from what I recall being told about some of the thinking behind this lessons. Or the assumption that ppl might have gone on a break, and are starting 'fresh' again. You're not wrong, I just didn't think to mess with that.

Reply:
Chrissy can be the tie-breaker, but I think we should try dropping it.

Reply:
I mean @chrissyrissmeyer

Reply:
Yeah that's fine. I think the nice thing here is the transition from using HEAD to integrating the `~1` syntax.

Reply:
We could look into introducing HEAD in the previous section.

Reply:
But at that point we might have to split up section 3 into two again.

Reply:
Yeah, I guess my vote is to leave it alone. Glossing over it quickly isn't an issue I think as it's really only a couple commands, and it's less potential refactoring or touching of other lessons. This is mostly out of laziness at this point, I admit..

Reply:
D:

Reply:
My general feeling is that it is good to remind folks of commands that they should be using regularly (e.g., git status )or at certain points, but not explain it again unless there is something in the new section that it amplifies or clarifies.

Reply:
Ok, I removed the initial git diff check. I still think the new line needs to be added to the README to actually demo the diff's, so I've left that in. The previous section does introduce `diff` and `diff --staged`, so I think this now carries on from where that leaves off (diff'ing against previous commits. Is this better, or was there more you all were hoping to drop in this section?

Reply:
We don't need the new line to show diffs like `HEAD~1..HEAD`, and we've already demonstrated diffing between HEAD and the uncommitted changes.